packages/aws-cdk/lib/api/hotswap-deployments.ts

```diff
@@ -50,7 +52,9 @@ export async function tryHotswapDeployment(
   }
 
   // apply the short-circuitable changes
+  print(`\n${ICON} %s\n`, colors.bold('starting hotswap deployment...'));
   await applyAllHotswappableChanges(sdk, hotswappableChanges);
+  print(`\n${ICON} %s\n`, colors.bold('hotswap deployment complete'));
```

Comment:
I feel like this is a little much...? I think a message per hotswapped resource is enough, I don't think we need these extra 'start' and 'end' messages.